.circleci/config.yml

```diff
@@ -36,6 +36,31 @@ aliases:
       command: |
         sudo apt-get update
         sudo apt-get install -y libcairo2-dev
+  - &check_versions
+    run:
+      name: Check that all versions are the same if $CIRCLE_TAG is defined
+      command: |
+        if [[ ! -z ${CIRCLE_TAG+x} ]]; then
+          python -m pip install --upgrade setuptools_scm
+          BACKEND_VERSION=$(python -m setuptools_scm)
+          echo "richie version: ${BACKEND_VERSION}"
+          # In the frontend
+          grep "\"version\": \"${BACKEND_VERSION}\",$" src/frontend/package.json
+          # In the cookiecutter site template backend
+          cat cookiecutter/{{cookiecutter.organization}}-richie-site-factory/template/{{cookiecutter.site}}/requirements/base.txt | grep "richie==${BACKEND_VERSION}$"
+          # In the cookiecutter site template frontend
+          cat cookiecutter/{{cookiecutter.organization}}-richie-site-factory/template/{{cookiecutter.site}}/src/frontend/package.json | grep "\"richie-education\": \"${BACKEND_VERSION}\"$"
+          # In the cookiecutter documentation
+          cat docs/cookiecutter.md | grep "cookiecutter gh:openfun/richie --directory cookiecutter  --checkout v${BACKEND_VERSION}"
+          cat docs/cookiecutter.md | grep "fundocker/cookiecutter gh:openfun/richie --directory cookiecutter --checkout v${BACKEND_VERSION}"
+          # In the e2e tests
+          cat tests_e2e/package.json | grep "\"version\": \"${BACKEND_VERSION}\",$"
+          # In the website
+          cat website/package.json | grep "\"version\": \"${BACKEND_VERSION}\",$"
+          # In the website versions
+          cat website/versions.json | grep "\"${BACKEND_VERSION}\",$"
+        fi
```

Comment:
After some thoughts, I think we could keep a tag format for release which fits the ont required by npm. Then `setuptools_scm` will be able to generate a version compliant with PEP440.

That's mean, the alias `check_versions` should be reworked as `BACKEND_VERSION` and `FRONTEND_VERSION` will not be the same.